irods-iBridgesCli.py

```diff
@@ -310,24 +329,27 @@ def main(argv):
         print(BLUE+'Upload complete with the following parameters:')
         print(json.dumps(config, indent=4))
         print(DEFAULT)
+        ic.session.cleanup()
     elif operation == 'download':
-        print(json.dumps(config, indent=4))
+        #print(json.dumps(config, indent=4))
         if prepareDownload(irodsPath, ic, config):
             downloadDir = config['DOWNLOAD']['path']
             irodsDataPath = config["iRODS"]["downloadItem"]
             print(YEL, 
-                  'Downloading: '+irodsDataPath+', '+str(ic.get_irods_size([irodsDataPath])/1024**3)+'GiB',
+                  'Downloading: '+irodsDataPath+', '+str(ic.get_irods_size([irodsDataPath])/1000**3)+'GB', 
```

Comment:
The units should be consistent throughout.  Has an issue been created for this?  I don't see one.

Reply:
No we still have to agree on a unit. The iRODS server convention seems to be GiB, however users wil understand GB better. So for the client I would opt for GB. But Indeed I still need to check the whole code base for this.

Reply:
Yeah, this is a major pet peeve of mine (can you tell?).

I _strongly_ prefer binary units, but absolutely _hate_ their names.  The binary units were first and the decimal should have the `i` in the name!

It doesn't help that HDD/SSD manufacturers have stuck to decimal units since it gave them a marketing advantage decades ago, but RAM manufacturers have stuck to binary units because of manufacturing and other technical constraints.  There is also a divide in the unit defaults for various tools (e.g., fdisk, du, df, ls, etc.).

Generally, however, units in RAM tend to be binary and units on disk tend to be decimal.  Given that, the official names, and what a general user will expect, decimal GB seems the most ideal.  The non-technical users won't really care but will also not be confused.  The technical users will understand despite any annoyance ;)

Reply:
I have made everything consistently decimal units as this seems to make the most sense given the audience.  We can always change it to binary units with minimal effort.